v2/cli/init_templates.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for a non-git project
+// nolint: lll
+var projectTemplate = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ProjectID }}
+description: My new Brigade project
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/cli
+    types:
+    - exec
```

Comment:
Let's comment this section and treat it as an example. Let's also add more comments above to explain the example.

Reply:
I might be misunderstanding your suggestion. Do you mean I should leave this section commented out like so? Also I'm not sure if my description is accurate.
```
spec:
  # This section defines any arbitrary event for your project to listen
  # and respond to.
  # eventSubscriptions:
  # - source: brigade.sh/cli
  #   types:
  #   - exec
```

Reply:
Try this:

```
## Subscribe below to any events that should trigger your script.
## The commented example depicts a subscription to "exec"
## events originating from the Brigade CLI. 
# eventSubscriptions:
# - source: brigade.sh/cli
#   types:
#   - exec
```

Note that the double # are deliberate to distinguish between a comment that is _commentary_ and the comments below it which are commented YAML. This is a widely used convention in the Kubernetes community.